operations/multi_tenancy/Modify_a_Tenant.md

```diff
@@ -0,0 +1,122 @@
+# Modifying a tenant
+
+This page provides information about how to modify a tenant.  Modifications that are supported are limited to updating the list of `xnames` assigned to this tenant, as well as adding/deleting `childNamespaces`.
+
+## Table of Contents
+
+- [Modifying a tenant](#modifying-a-tenant)
+  - [Table of Contents](#table-of-contents)
+  - [Modify the existing TAPMS Custom Resource (CR)](#modify-the-existing-tapms-custom-resource-cr)
+  - [Apply the modified TAPMS Custom Resource (CR)](#apply-the-modified-tapms-custom-resource-cr)
+  - [Modify the Slurm Operator Custom Resource (CR)](#modify-the-slurm-operator-custom-resource-cr)
+  - [Apply the Slurm Operator Custom Resource (CR)](#apply-the-slurm-operator-custom-resource-cr)
+
+## Modify the existing TAPMS Custom Resource (CR)
+
+Below is an example of a change to add an xname to a tenant:
+
+- (`ncn-mw#`)
+
+    ```bash
+    cat tenant.yaml
+    apiVersion: tapms.hpe.com/v1alpha1
+    kind: Tenant
+    metadata:
+      name: vcluster-blue
+    spec:
+      childnamespaces:
+        - user
+        - slurm
+      tenantname: vcluster-blue
+      tenantresources:
+        - type: compute
+          hsmgrouplabel: blue
+          enforceexclusivehsmgroups: true
+          xnames:
+            - x0c3s5b0n0
+            - x0c3s6b0n0
+            - x0c3s7b0n0 <-- Adding this xname
+    ```
+
+## Apply the modified TAPMS Custom Resource (CR)
+
+When a tenant CRD is applied, `tapms` will determine any changes to the tenant, and reconcile any changes to `childNamespaces` and `xnames`.
+
+- (`ncn-mw#`)
+
+    ```bash
+    kubectl apply -n tenants -f <tenant.yaml>
+    tenant.tapms.hpe.com/vcluster-blue configured
+    ```
+
+It can take up to a minute for `tapms` to reconcile the change.  The following command can be used to monitor the status of the tenant:
+
+- (`ncn-mw#`)
+
+    ```bash
+    kubectl get tenant -n tenants vcluster-blue -o yaml
+    apiVersion: tapms.hpe.com/v1alpha1
```

Comment:
```suggestion
- (`ncn-mw#`) It can take up to a minute for `tapms` to reconcile the change. The following command can be used to monitor the status of the tenant:

    ```bash
    kubectl get tenant -n tenants vcluster-blue -o yaml
    ```

    Example output:

    ```yaml
    apiVersion: tapms.hpe.com/v1alpha1
```